crates/oxc_linter/src/rules/unicorn/no_static_only_class.rs

```diff
@@ -0,0 +1,156 @@
+use oxc_ast::AstKind;
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error(
+    "eslint-plugin-unicorn(no-static-only-class): Disallow classes that only have static members."
+)]
+#[diagnostic(
+    severity(warning),
+    help("Change the class to an object instead, or add some instance members.")
```

Comment:
The message "A class with only static members could just be an object instead." feels like a better explanation?

Reply:
yeah, good point